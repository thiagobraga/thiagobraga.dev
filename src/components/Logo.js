import React, { Component } from 'react'
import '../styles/components/animation.sass'
import '../styles/components/logo.sass'

export default class Logo extends Component {
  render() {
    return (
      <span className={this.props.large ? 'logo large' : 'logo'}>
        Thiago <b>Braga</b>
      </span>
    )
  }
}
