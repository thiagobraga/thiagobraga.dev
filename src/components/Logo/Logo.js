import React, { Component } from 'react'
import '../../styles/components/animation.sass'
import '../../styles/components/logo.sass'

class Logo extends Component {
  render() {
    return (
      <span className={this.props.large ? 'logo large' : 'logo'}>
        Thiago <b>Braga</b>
      </span>
    )
  }
}

export default Logo
