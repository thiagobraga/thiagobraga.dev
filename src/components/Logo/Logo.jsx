import React from 'react';

import '../../styles/components/logo.sass';

export default function Logo(props) {
  const { size } = props;

  return (
    <span className={`logo ${size || ''}`.trim()}>
      Thiago <b>Braga</b>
    </span>
  );
}