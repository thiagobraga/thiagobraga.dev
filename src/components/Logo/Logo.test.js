import React from 'react'
import Logo from './Logo'
import { create } from 'react-test-renderer'

describe('Logo component', () => {
  test('It should render Thiago Braga', () => {
    const logo = create(<Logo/>)
    expect(logo.toJSON()).toMatchSnapshot()
  })
})
