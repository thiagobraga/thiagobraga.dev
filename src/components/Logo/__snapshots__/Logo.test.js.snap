// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component It should render Thiago Braga 1`] = `
<span
  className="logo"
>
  Thiago 
  <b>
    Braga
  </b>
</span>
`;
